content/docs/cli/environment-variables.md

```diff
@@ -111,6 +155,127 @@ aliases:
         </p>
         <pre><code class="text-xs">PULUMI_DEBUG_PROMISE_LEAKS=true</code></pre>
     </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_DEV
+        </span>
+    </dt>
+    <dd>
+        <p>
+            Enable features for hacking on pulumi itself.
+        </p>
+        <pre><code class="text-xs">PULUMI_DEV=true</code></pre>
+    </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_DISABLE_AUTOMATIC_PLUGIN_ACQUISITION
+        </span>
+    </dt>
+    <dd>
+        <p>
+            Disables the automatic installation of missing plugins.
+        </p>
+        <pre><code class="text-xs">PULUMI_DISABLE_AUTOMATIC_PLUGIN_ACQUISITION=true</code></pre>
+    </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_DISABLE_PROVIDER_PREVIEW
+        </span>
+    </dt>
+    <dd>
+        <p>
+            Disables provider preview and enabled previous more concervative preview behavior.
+        </p>
+        <pre><code class="text-xs">PULUMI_DISABLE_PROVIDER_PREVIEW=true</code></pre>
+    </dd>
+    <dt>
+        <span class="font-mono">
+            PULUMI_DISABLE_RESOURCE_REFERENCES
```

Comment:
I don't think we need this anymore, so not sure we should even document it. Rather, we should clean up its usage.